tools/c7n_mailer/c7n_mailer/record.py

```diff
@@ -1,150 +0,0 @@
-# Copyright 2016 Capital One Services, LLC
```

Comment:
there mostly display related wrt to a resource, minus the date utility you added, it would be helpful due to the size of this pr to not do unrelated renames as well. the intent was that the record formatter could go away given there's generic formatting in c7n core now.